mlir/test/Integration/Dialect/Arith/CPU/test-semantics.mlir

```diff
@@ -0,0 +1,52 @@
+// Regression test suite from an independent development of Arith's
+// semantics and a fuzzer
```

Comment:
Can you change this comment, to state that you are testing the results of the trunc/extsi/extui operations?
I would also not personally include that this comes from somewhere else, as this seems something that should be tested anyway?
Also, I would change the file name to something more descriptive (like test-lowering-arith-ext-trunc.mlir)

Reply:
> to state that you are testing the results of the trunc/extsi/extui operations?
These tests are a few that I picked from the top of the pile, there's more incoming - thanks for the suggestion to sort them by type ! 

> I would also not personally include that this comes from somewhere else, as this seems something that should be tested anyway?
yep fair enough! doesn't really matter where they came from at the end of the day :)